Documentation/ColumnsConfig/Type/Text/Index.rst

```diff
@@ -12,6 +12,9 @@ Text areas & RTE
     A developer does not need to define this field in an extension's
     :file:`ext_tables.sql` file.
 
+The :ref:`according database fields <t3coreapi:auto-generated-db-structure>`
+for all render types are generated automatically.
```

Comment:
This is the only place where you have written "all render types" instead of just "is generated automatically".

Reply:
We are on the type=text overview page where there are different sub rendertypes.